lunar/distribution.yaml

```diff
@@ -1323,7 +1323,11 @@ repositories:
       tags:
         release: release/lunar/{package}/{version}
       url: https://github.com/ktossell/libuvc-release.git
-      version: 0.0.5-3
+      version: 0.0.6-0
+    source:
+      type: git
+      url: https://github.com/ktossell/libuvc.git
+      version: 0.0.6
```

Comment:
the version of the source entry needs to be a branch and not a tag. In this case `master` seems to be the appropriate one

Reply:
Thanks! I wasn't sure the meaning.